buildSrc/src/main/java/org/opensearch/gradle/docker/DockerSupportService.java

```diff
@@ -66,13 +67,17 @@ public abstract class DockerSupportService implements BuildService<DockerSupport
 
     private static Logger LOGGER = Logging.getLogger(DockerSupportService.class);
     // Defines the possible locations of the Docker CLI. These will be searched in order.
-    private static String[] DOCKER_BINARIES_UNIX = { "/usr/bin/docker", "/usr/local/bin/docker" };
+    private static String[] LINUX_BIN_LOCATIONS = { "/usr/local/bin", "/usr/bin", "/opt/homebrew/bin", "/home/linuxbrew/.linuxbrew/bin" };
```

Comment:
`"/home/linuxbrew/.linuxbrew/bin"` this one is unclear: basically we are pointing to the home folder of the `linuxbrew` user which basically should fail the lookup unless run under `linuxbrew` user or `root`. Or I am missing something?

Reply:
I'm not sure anything will fail here unless the Docker CLI can't be found at all? I think like the `PATH` env variable it will search folders even if they don't exist and only throw an error if the binaries are not present? Might verify? 

Aside from that technical nuance, I think I agree that we should leave the ~linuxbrew user out of the path here. Users can always create a `/usr/local/bin` softlink to the linuxbrew home directory if this is their configuration.

Reply:
Thanks @nknize , I think "fail" is indeed too strong, I would rephrase that as "unsuccessful lookup" since we attaching to particular user, thank you.

Reply:
> Perhaps we should add one or two debug messages here as well?

Debug verbosity here would be good! Anything to help users understand what's going on is a great thing. I'll keep an eye on the PR and review when ready.